how-to-guides/re-run-failed-training/scripts/re_run_failed_training.py

```diff
@@ -0,0 +1,113 @@
+import neptune.new as neptune
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import os
+from torchvision import datasets, transforms
+
+# Step 1: Get Run ID
+
+# Fetch project
+project = neptune.get_project(name='common/showroom', api_token='ANONYMOUS')
+
+# Fetch only inactive runs
+runs_table_df = project.fetch_runs_table(state="idle", tag=['showcase-run']).to_pandas()
+
+# Sort runs by failed 
+runs_table_df = runs_table_df.sort_values(by='sys/failed', ascending=True)
+
+# Extract the last failed run's id
+failed_run_id = runs_table_df[runs_table_df['sys/failed']==True]['sys/id'].values[0]
+
+print('Failed_run_id = ', failed_run_id)
+
+# Step 2: Resume failed run
+failed_run = neptune.init(
+    project="common/showroom",
+    api_token="ANONYMOUS",
+    mode="read-only",
+    run=failed_run_id
+)
+
+# Step 3: Fetching and downloading data from Neptune
```

Comment:
In the previous steps you use Present simple and here you use Present continuous. Please use only simple form -> drop `...ing` from everywhere in script and notebook.